src/graph_print.cc

```diff
@@ -128,10 +128,18 @@ void Graph::print_includes(std::ostream &dst)
 	dst << "#include <stdbool.h>" << std::endl;
 	dst << "#include <stdint.h>" << std::endl;
 	dst << "#include <string.h>" << std::endl;
+	dst << std::endl;
 
 	dst << "#define MAX(X,Y) ( X > Y ? X : Y)" << std::endl;
 	dst << "#define MIN(X,Y) ( X < Y ? X : Y)" << std::endl;
 	dst << "#define CLIP(X,L) ( MAX(MIN(X,L), -L) )" << std::endl;
+	dst << std::endl;
+
+	dst << "#ifdef _MSC_VER" << std::endl;
```

Comment:
```suggestion
 	// 'inline' functions are a C99 addition.
	dst << "#if __STDC_VERSION__ < 199901L" << std::endl;
```
Would this work for you?

Reply:
Works for me also. The more important point to me is the ability to change the prefix from a central location.

Reply:
Nice.
Could you amend these changes to your commit, I'll then rebase-merge your branch.
I'd rather keep the two commits separate and not squash.